docs/links-app.md

```diff
@@ -0,0 +1,25 @@
+# The Links app
+
+The Links module provides a content model named "Link" which allows you to have a navigation item without any page content associated;
+instead, its navigation entry will redirect to an arbitrary URL.
+
+## Configuration
+
+Ensure both `cms.apps.pages` and `cms.apps.links` are in your project's `INSTALLED_APPS`. If they weren't already, you will need to migrate:
+
+```
+$ ./manage.py migrate
+```
+
+## Usage
+
+To add a Link to your site's navigation menu, add a Page and select the "Link" page type.
```

Comment:
backticks?

Reply:
What was actually wrong, was that this should have been just "link". :)